preparation.py

```diff
@@ -60,3 +77,12 @@ def preprocess_image(img, target_size=(128, 128)):
     else:
         x = torch.autograd.Variable(x, volatile=True)
     return x
+
+
+def add_dropped_objects(original, processed):
+    reconstructed = processed
+    labeled = label(original)
+    for i in range(1, labeled.max() + 1):
+        if np.sum((labeled == i) * processed) == 0:
+            reconstructed += (labeled == i).astype('uint8')
```

Comment:
@apyskir I think casting to uint8 once at the very end shold suffice

Reply:
@jakubczakon Yes, it will be casted only once.